src/components/product/ProductCard/ProductCard.tsx

```diff
@@ -59,10 +59,10 @@ function ProductCard({
 
   return (
     <UICard
-      className="product-card"
-      data-card-variant={variant}
-      data-card-bordered={bordered}
+      data-fs-product-card-variant={variant}
+      data-fs-product-card-bordered={bordered}
       data-card-out-of-stock={outOfStock}
```

Comment:
I think we can remove this data attr, we are not using it anymore and so far we don't need it here.

![image](https://user-images.githubusercontent.com/3356699/160211976-1fd7ff58-72cb-43c0-a19f-d8a81d9be969.png)







```suggestion
```